articles/microsites/add-login/add-login-regular-web-app.md

```diff
@@ -33,23 +33,13 @@ For security in server-side web apps, Auth0 uses the [Regular Web App Login Flow
 ## Implementation overview
 
 ::: steps
-  1. <strong>Configure the sign-in methods</strong><br/><br/>Auth0 supports a wide range of authentication methods: regular username/password (users can be stored in Auth0 or your own database), social (i.e., Google, Facebook, and 50+ other providers), passwordless (email magic link, email code, and phone code), and enterprise (e.g., SAML-based, ADFS, Ping, Okta).<br/><br/>Go to the dashboard and turn on the methods you want to allow; they will automatically show up in the login/sign-up page. By default, email/password and Google are enabled.
+  1. <h4>Configure the sign-in methods</h4>Auth0 supports a wide range of authentication methods: regular username/password (users can be stored in Auth0 or your own database), social (i.e., Google, Facebook, and 50+ other providers), passwordless (email magic link, email code, and phone code), and enterprise (e.g., SAML-based, ADFS, Ping, Okta).<br/>Go to the dashboard and turn on the methods you want to allow; they will automatically show up in the login/sign-up page. By default, email/password and Google are enabled.
 
-  2. <strong>Customize the sign-in UI (optional)</strong><br/><br/>The default experience is demonstrated in the image below and can be completely customized in the dashboard, from changing the logo and primary colors to completely overriding it with your own login screen.<br/><br/><img src="/media/articles/microsites/login-screen-default-web.png" alt="Default Login Screen for Native/Mobile Apps" width="70%">
-
-  3. <strong>Use the Auth0 SDK to trigger the flow</strong><br/><br/>The SDK will take care of the details of opening the SafariViewController or Chrome Custom Tab, parsing the response back from Auth0, and validating the ID Token.<br/><br/>Your app can store the Access Token and a Refresh Token used to renew the Access Token without asking the user to re-enter their credentials. Follow one of our [Regular Web App Quickstarts](/quickstart/webapp) to get started with the integration.
+  2. <h4>Customize the sign-in UI (optional)</h4>The default experience is demonstrated in the image below and can be completely customized in the dashboard, from changing the logo and primary colors to completely overriding it with your own login screen.<br/><img src="/media/articles/microsites/login-screen-default-web.png" alt="Default Login Screen for Native/Mobile Apps" width="70%">
 
+  3. <h4>Use an OpenID Connect SDK to trigger the flow</h4>The SDK will redirect to the Auth0 Universal login page and handle the response validating the ID Token.<br/>Your app can store the ID Token. Follow one of our <a href="/quickstart/webapp">Regular Web App Quickstarts</a> to get started with the integration.
```

Comment:
Propose changing "The SDK will redirect to the Auth0 Universal login page" to "The SDK will redirect to our hosted universal login page". The second is how we refer to it earlier as it's a general concept as opposed to the Auth0 name for the thing, which is Universal Login. So it would otherwise technically need to be "the Auth0 Universal Login login page", which is a mouthful and not particularly descriptive for a newbie.

Reply:
I think that most places in the docs, we say "the login page" and when required "the Universal Login page" - not saying those are better options, just reminding of patterns. I think the reason we try really hard not to use "hosted" is because of the terminology shift from Hosted Login Page -> Universal Login (even though that was never a 1:1 concept).

Reply:
Good to know. We're just trying to specify that you're coming to _our_ login page, not some login page you have decided to implement yourself. And we haven't really introduced the name "Universal Login" as such yet because of the basic audience level. Hmm.

Reply:
Yeah, the point about "the login page" not being specific enough is one I've also pointed out, but nevertheless it's what we use a lot - not a problem unique to this set of docs unfortunately. Like I said, it stemmed from people being so resistant to the "hosted" term (despite it being well known by current customers). 

In this specific case, you're already using the word "universal login" but now just adding "hosted" to it, and if this is directed at new customers I can't imagine any of that makes sense at all.

Maybe if we haven't introduced the concept of Universal Login at this point, we're doing something wrong. We're already basically explaining how to use Universal Login here, at least in a summarized way, so maybe we should be taking a line to define it before we start? WDYT?

Reply:
Maybe we could define Universal in a roundabout way, like "which we call Universal Login" at the first mention. Might still be easier to keep it general in which case, "Auth0-hosted default login page"? I don't know. Seems unwieldy, but more clear than the others?